test/priors/test_lkj_prior.py

```diff
@@ -46,11 +44,12 @@ def test_lkj_prior_log_prob_cuda(self):
     def test_lkj_prior_batch_log_prob(self, cuda=False):
         device = torch.device("cuda") if cuda else torch.device("cpu")
         prior = LKJPrior(2, torch.tensor([0.5, 1.5], device=device))
+        dist = LKJCholesky(2, torch.tensor([0.5, 1.5], device=device))
 
         S = torch.eye(2, device=device)
-        self.assertTrue(approx_equal(prior.log_prob(S), torch.tensor([-1.86942, -0.483129], device=S.device)))
+        self.assertTrue(approx_equal(prior.log_prob(S), dist.log_prob(S.cholesky())))
```

Comment:
`.cholesky` will raise a `DeprecationWarning`. Can you use `torch.linalg.cholesky` instead?